src/core.ttl

```diff
@@ -191,11 +191,12 @@ _:norman2013 dct:bibliographicCitation "Norman, D. (2013). The design of everyda
 
 :Workspace a owl:Class ;
         rdfs:isDefinedBy :core ;
-        rdfs:comment "A container for one or multiple interactions among agents and potentially artifacts. Workspaces can contain any resource in general and in particular agents, artifacts, or other workspaces. A workspace can be a Linked Data Platform container and this can be done by multi-typing the container or by multiple inheritance on a class ex:Workcontainer."@en ;
+        rdfs:comment "A logical container for any resource in general and in particular for agents, artifacts, platforms or other workspaces. Workspaces can be created or deleted by agents on a needed basis and all agent interactions happen in the context a workspace whether or not it was explicitly reified. A workspace can be, for instance, implemented on top of an LDP indirect container."@en ;
```

Comment:
@FabienGandon thanks for the updated definition! I propose to remove platforms from the list of resources that can be included in a workspace. A platform could be accessed through an artifact that is contained in a workspace, but otherwise, a platform would be the supporting infrastructure for the workspace.

Reply:
Done.

Reply:
@andreiciortea the change was done.
@Antoine-Zimmermann and @DrLeturc please review ASAP